docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -1,7 +1,9 @@
 [[get-started-elastic-stack]]
 == Getting started with the {stack}
 
-Looking for an {stack} ("ELK") guide that shows how to quickly install and configure the {stack}? You're in the right place! You can install the {stack} on a single VM, or even on your laptop:
+Looking for a guide that shows how to quickly install and configure the {stack}?
```

Comment:
Can you check with @AnneB-SEO  to see if users are still searching for ELK tutorials? (We added this awhile ago because users were finding 3rd party instructions rather than our docs.)

Reply:
Talked to Anne, and ELK > Elastic Stack. Thanks for pointing this out! I'll add "ELK" back in.